src/pages/Proposal/VoteSummary/index.tsx

```diff
@@ -0,0 +1,94 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { BigNumber } from 'bignumber.js';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { generateBscScanUrl } from 'utilities';
+import { useTranslation } from 'translation';
+import { XVS_TOKEN_ID } from 'constants/xvs';
+import { Button, Icon, LabeledInlineContent, EllipseText, Tooltip } from 'components';
+import { ActiveVotingProgress } from 'components/v2/GovernanceProposal/ActiveVotingProgress';
+
+import { useStyles } from './styles';
+
+type VoteFrom = {
+  address: string;
+  voteWeightWei: BigNumber;
+  comment?: string;
+};
+
+type VoteType = 'for' | 'against' | 'abstain';
+
+interface IVoteSummaryProps {
+  onClick: () => void;
+  voteType: VoteType;
+  votedValueWei?: BigNumber;
+  votedTotalWei?: BigNumber;
+  votesFrom?: VoteFrom[];
+  className?: string;
+  isDisabled?: boolean;
+}
+
+export const VoteSummary = ({
+  onClick,
+  voteType,
+  votedTotalWei = new BigNumber(0),
+  votedValueWei = new BigNumber(0),
+  votesFrom = [],
+  className,
+  isDisabled,
+}: IVoteSummaryProps) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+
+  return (
+    <Paper css={styles.root} className={className}>
+      <ActiveVotingProgress
```

Comment:
This should be a `LabledProgressBar`

Reply:
It seems to me that handling texts and colors on the ActiveVotingProgress side should avoid us from duplicating this logic as we need it in several places